AsyncDisplayKit/ASNavigationController.h

```diff
@@ -16,6 +16,14 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ * ASNavigationController
+ *
+ * @discussion ASNavigationController is a drop in replacement for UINavigationController
+ * which implements the memory efficiency improving @c ASManagesChildVisibilityDepth protocol.
```

Comment:
Add another line with something like: You can use ASNavigationController with regular UIViewControllers, as well as ASViewControllers.  It is safe to subclass or use even where AsyncDisplayKit is not adopted.